lib/bovine/interface.py

```diff
@@ -22,26 +23,45 @@ def get_all(self):
   def search(self, inv_type, keyword):
     """Search the inventory
```

Comment:
we should make it standard practice to document the expected input/output of a function/method. 

here are two great examples:

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
```
    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.
```

http://docs.python-guide.org/en/latest/writing/documentation/
```
    Summary line.

    Extended description of function.

    Parameters
    ----------
    arg1 : int
        Description of arg1
    arg2 : str
        Description of arg2

    Returns
    -------
    int
        Description of return value
```